src/utils/dom.ts

```diff
@@ -162,15 +162,28 @@ export async function focusElement(el: FocusableElement): Promise<void> {
   return isCalciteFocusable(el) ? el.setFocus() : el.focus();
 }
 
+function getAssignedElements<T extends Element = Element>(element: Element, slotName: string | string[]): T[] {
+  const slotSelector = Array.isArray(slotName)
+    ? slotName.map((name) => `slot[name="${name}"]`).join(",")
```

Comment:
I wonder if we should support multiple slot queries at once. We might be able to get rid of this support.